### PR TITLE
uefi-run: init at 0.5.0

### PR DESCRIPTION
**Pull request by ghost**
###### Description of changes

A little helper tool for uefi osdev that builds the esp image without need for root
https://github.com/Richard-W/uefi-run

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -7834,6 +7834,12 @@
     githubId = 42545625;
     name = "Maas Lalani";
   };
+  maddiethecafebabe = {
```
Comment (misuzu) [inline maintainers/maintainer-list.nix:7837]: Please extract this change to a separate commit, see https://github.com/NixOS/nixpkgs/pull/183559 for an example.

**Comment by maddiethecafebabe** [comment]
Alright thanks, is this correct?

**Review by misuzu — APPROVED** [review]
> Alright thanks, is this correct?

Yes, LGTM!

**Comment by maddiethecafebabe** [comment]
Do i still have to fix something/is something wrong?

File: pkgs/tools/virtualization/uefi-run/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+rustPlatform.buildRustPackage rec {
+  pname = "uefi-run";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "Richard-W";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-fwzWdOinW/ECVI/65pPB1shxPdl2nZThAqlg8wlWg/g=";
+    name = "${pname}-git-v${version}";
```
Comment (SuperSandro2000) [inline pkgs/tools/virtualization/uefi-run/default.nix:15]:
```suggestion
```
